### DISCUSSION TITLE
Is there a faster speed? Such as TensorRT? has anyone tried

### DISCUSSION THREAD

**Discussion by standyyyy** [SYSTRAN/faster-whisper#134] [Ideas]
I am a novice, and I am very interested in ASR recently. In whisperx, I saw that the author used an unopen source method to improve whisper to translate one hour of audio in one minute. I am very enthusiastic about this. I am trying to use TensorRT to speed up whisper. Reasoning has not been successful for the time being. Are you interested? Has anyone tried it?

**Comment by phineas-pta** [comment]
with `whisperx` the author stated

> API with batched inference (transcribe 1hr audio in under 1min)

that method isn't a secret, you can find an open source implementation here: openai/whisper#662

i didn't know anything about TensorRT but it's discussed here: openai/whisper#169
